libs/wire-api/src/Wire/API/Routes/Public/Galley/Messaging.hs

```diff
@@ -152,8 +154,7 @@ data MessageNotSent a
   deriving
     (AsUnion (MessageNotSentResponses a))
     via (GenericAsUnion (MessageNotSentResponses a) (MessageNotSent a))
-
-instance GSOP.Generic (MessageNotSent a)
+  deriving anyclass GSOP.Generic
```

Comment:
why is this better?

Reply:
it's equivalent but less code.

Reply:
there's actually a difference in general, namely if you use the `deriving X` directly underneath the type, GHC tries to insert the constraints on the derived classes itself which is not the case when doing `deriving instance` or `instance X where` where you have to specify the context yourself